src/components/info-tile/info-tile.tsx

```diff
@@ -226,8 +226,8 @@ export class InfoTile {
                     class="progress"
                     prefix={this.progress?.prefix}
                     value={this.progress.value}
-                    suffix={this.progress.suffix}
-                    maxValue={this.progress.maxValue}
+                    suffix={this.progress?.suffix}
+                    maxValue={this.progress?.maxValue}
```

Comment:
These question marks shouldn't be necessary, because we've wrapped this in an `if` that checks that `this.progress.value` exists. If `this.progress.value` exists, clearly `this.progress` must exist 😄

Reply:
ah, okay.  👍 I will add a fixup :)